### DISCUSSION TITLE
Cannot edit or move cursor inside input

### DISCUSSION THREAD

**Discussion by HrideyReact** [csandman/chakra-react-select#237] [Q&A]
### Description

In the current behaviour, I cannot edit the text or move the cursor to see the whole text, the cursor is stuck on the start of the selected value, any clue how we can overcome this behaviour?

![Screenshot 2023-01-06 at 5 22 24 PM](https://user-images.githubusercontent.com/92137808/211007273-abd0cb59-80ac-4e88-a2de-b90c79016b6d.png)

### chakra-react-select Version

4.3.0

### Link to Reproduction

https://codesandbox.io/s/chakra-react-select-border-colors-forked-iort31?file=/data/data.js

### TypeScript?

- [X] Yes I use TypeScript

### Steps to reproduce

You can reproduce this on any component.

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS/iPadOS
- [ ] Android

**Comment by csandman** [comment]
Sorry for the late response on this, but there's not much that can be done about it.  Once a value is selected, it is no longer actively in the input. It is just there so you can see the selected value. If you start typing again, it will start searching for a new option. This is how `react-select` itself works.

The only thing I could recommend is making your select large enough so you can see an entire selected option. And in the case of a creatable select, make it larger than you'd expect most entered to be.
